src/app/auth/index.ts

```diff
@@ -0,0 +1,18 @@
+import { extendSession, listIdentities } from "@services/kratos"
+
+// TODO: interface/type should be reworled so that it doesn't have to come from private
+import { listSessionsInternal } from "@services/kratos/private"
```

Comment:
This is what `private` is meant for. To be used internally to the `auth` module and not `export`ed... looks fine to me.

Reply:
it's exported to the use case here. so outside the service. does that still looks good for you?